src/lib.rs

```diff
@@ -218,10 +218,10 @@ pub trait OpenDrainOutput {
     type Error: Sized + Debug;
 
     /// Is the input pin high?
-    fn is_high(&self) -> Result<bool, Self::Error>;
+    fn is_high(&mut self) -> Result<bool, Self::Error>;
```

Comment:
Hmm, mut here as well as in PR20

Reply:
Jep embedded-hal traits need `&mut` even for reading Input https://github.com/rust-embedded/embedded-hal/blob/master/embedded-hal/src/digital.rs#L208